project/Boilerplate.scala

```diff
@@ -222,21 +222,6 @@ object Boilerplate {
       -   final def andThen[H[${`_.._`}]](other: PolyFunction$suffix[G, H]): PolyFunction$suffix[F, H] = new PolyFunction$suffix[F, H]{
       -      def apply[${`A..N`}](fa: F[${`A..N`}]): H[${`A..N`}] = other(self(fa))
       -   }
-      -
-      -   import Kind$arity._
```

Comment:
removing this because the unsafe caching can be backed by maps or arrays depending on the situation